internal/runners/auth/auth.go

```diff
@@ -68,27 +88,24 @@ func (a *Auth) Run(params *AuthParams) error {
 }
 
 func (a *Auth) authenticate(params *AuthParams) error {
-	if params.Token == "" {
-		var err error
-		if params.Interactive || params.Username != "" {
-			err = authlet.AuthenticateWithInput(params.Username, params.Password, params.Totp, a.Cfg, a.Outputer, a.Prompter, a.Auth)
-		} else {
-			err = authlet.AuthenticateWithBrowser(a.Outputer, a.Auth, a.Prompter)
-		}
-		if err != nil {
-			return locale.WrapError(err, "login_err_auth")
-		}
-	} else {
-		err := a.Auth.AuthenticateWithModel(&mono_models.Credentials{
+	if params.Interactive || params.Username != "" || params.Totp != "" {
```

Comment:
Don't think we need totp here, the `verify` step should catch that.

Totp refers to "two factor auth". It is useless if you're not also supplying the username and password.

Reply:
Thanks! I was uncertain about that flag.